habushu-maven-plugin/src/test/resources/specifications/containerize-dependencies.feature

```diff
@@ -0,0 +1,12 @@
+@containerizeDependencies
+Feature: Test staging source files of specified dependencies and transitive path-based dependencies
+
+  Scenario: One Habushu-type dependency is specified with the default configuration
+    Given a single Habushu-type dependency
```

Comment:
S: I think language here is very important to make it crisp.  I think we are saying:
```gherkin
Given a single dependency with a packaging type of habushu
...
Then the source files of the dependency and its associated transitive Habushu packaged dependencies are staged for containerization
```

Reply:
updated, thank you for the suggest!